docs/design/testing.md

```
@@ -16,7 +20,7 @@ The performance list in `AGENTS.md` defines the protected scope. The proof sourc
 - The [input note](input.md) states the poll-path synchronization.
 - The [logging note](logging.md) states the queue and sink mechanisms.
 - The [memory note](memory-scanning.md) states the cache and guarded-memory mechanisms.
-- [`rtti.hpp`](../../include/DetourModKit/rtti.hpp) states the allocation and read bounds for `vtable_is_type`.
+- [ `rtti.hpp` ](../../include/DetourModKit/rtti.hpp) states the allocation and read bounds for `vtable_is_type`.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)